web/vtadmin/src/components/routes/Tablets.tsx

```diff
@@ -35,6 +35,12 @@ import { TabletLink } from '../links/TabletLink';
 import { ExternalTabletLink } from '../links/ExternalTabletLink';
 import { ShardLink } from '../links/ShardLink';
 import InfoDropdown from './tablets/InfoDropdown';
+import { isReadOnlyMode } from '../../util/env';
```

Comment:
This is not necessary in this PR, but it would be cool if we had a "ReadOnlyGate" component that did all the checking of the `isReadOnlyMode` env variable for us, like:

```
<ReadOnlyGate>
    <DataCell>
         <InfoDropdown alias={t.alias as string} clusterID={t._raw.cluster?.id as string} />
     </DataCell>
</ReadOnlyGate>
```

and `ReadOnlyGate` handles the repeated logic of conditional rendering 😄  but it might be overkill, I love gates!

Reply:
Hahaha Frances! I was thinking of adding a `ReadOnlyGate` component but I couldn't decide on a name. 😆 If you don't mind waiting a lil bit to rebase, I can add that real fast in this PR.

Reply:
I'll definitely wait for the gate! We stan a good gate!

Reply:
Hahaha, I added it! Thank you for the suggestion.

Reply:
Argh these tests are taking forever! 👹 Will merge ASAP!

Reply:
🥲